test/kubernetes_basic_example_logs_test.go

```diff
@@ -0,0 +1,91 @@
+// NOTE: we have build tags to differentiate kubernetes tests from non-kubernetes tests. This is done because minikube
```

Comment:
Should be added tags to run tests in Kubernetes context, tests are failing now
```
//go:build kubeall || kubernetes
// +build kubeall kubernetes
```

https://github.com/gruntwork-io/terratest/blob/d796b1dc789e373596c579d64567751c69fabdff/test/kubernetes_basic_example_test.go#L1

```
Failed
TestKubernetesBasicExampleLogsCheckWithContainerName 2023-01-31T18:44:36Z client.go:42: Configuring Kubernetes client using config file /home/circleci/.kube/config with context 
TestKubernetesBasicExampleLogsCheckWithContainerName 2023-01-31T18:44:36Z client.go:46: Error loading api client config, falling back to in-cluster authentication via serviceaccount token: stat /home/circleci/.kube/config: no such file or directory
    namespace.go:15:
```